.travis.yml

```diff
@@ -22,3 +22,13 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
+
+env:
+  -CODACY_PROJECT_TOKEN=bac08ffda9f94be6b07f52277efe2794
```

Comment:
Is it safe for us to put the project token here? 🤔

Reply:
Yes